src/saga.js

```diff
@@ -11,10 +10,9 @@ export function* fetchSaga({
       credentials: 'include',
       ...fetchOptions
     });
-    const next = yield call(Promise.resolve.bind(Promise), success(response));
-    yield put(next);
+    yield put(yield call(success, response));
```

Comment:
does this actually work? i had issues when `success` returned a non promise 🤷‍♂️

Reply:
i would think so: https://redux-saga.js.org/docs/api/index.html#callfn-args

> fn: Function - A Generator function, or normal function which either returns a Promise as result, or any other value.

Reply:
maybe redux-saga added support for it now 👍 if the tests pass then 🤷‍♂️